### PR TITLE
Web muhammed

### PR DESCRIPTION
**Pull request by gktpmuhammed**
Landing page is ready. Please review and give feedback.

### CODE REVIEW

**Comment by emiroztoprak** [comment]
All the changes seem to be working correctly. I think it is ready to merge.

File: Web/src/screens/bootstrap.js [hunk 1]
```diff
@@ -2,10 +2,11 @@
 import React, {useState} from 'react';
 import {Link} from 'react-router-dom';
 import {urls} from 'DATABASE';
-
-import {Dropdown,ButtonGroup,Alert,Accordion,Button, DropdownButton,Spinner,Card, ProgressBar, Carousel} from 'react-bootstrap';
 import faker from 'faker/locale/tr';
 import { useSelector } from 'react-redux';
+import SideBar from 'components/navbar/SideBar';
```
Comment (4teko7) [inline Web/src/screens/bootstrap.js:7]: You should remove SideBar from bootstrap screen. It is already in navigation.

File: Web/src/screens/bootstrap.js [hunk 2]
```diff
@@ -39,7 +38,9 @@ const Bootstrap = (props) => {
   <button type="button" class="btn btn-primary">Middle</button>
   <button type="button" class="btn btn-primary">Right</button>
 </div>
-
+<div>
+<SideBar />
```
Comment (4teko7) [inline Web/src/screens/bootstrap.js:42]: This component is not necessary

File: Web/src/components/SignUpForm.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import App from 'App';
+import React from 'react'
+import "../App.css"
+
+
+function Form() {
```
Comment (4teko7) [inline Web/src/components/SignUpForm.js:6]: We are using React-Bootstrap. You should use components of React-Bootstrap. There are tons of components there. Currentlyi you are using normal html tags not components.

File: Web/src/components/LoginForm.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import App from 'App';
+import React from 'react'
+import "../App.css"
+import { useHistory } from "react-router-dom";
+
+function Form() {
```
Comment (4teko7) [inline Web/src/components/LoginForm.js:6]: We are using React-Bootstrap. You should use components of React-Bootstrap. There are tons of components there. Currentlyi you are using normal html tags not components.

File: Web/src/App.css [hunk 5]
```diff
@@ -36,3 +36,160 @@
     transform: rotate(360deg);
   }
 }
+
```
Comment (4teko7) [inline Web/src/App.css:39]: Don't put lots of css in App.css
This file is valid for all pages. You should use component based css files. I mean, form.css login.css etc. can be created for some components. React-Bootstrap gives you lots of flexibility. You don't need to you lots of css.

File: Web/src/screens/landingPage.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+      
+      </div>
+      <div>
+      <Card style={{ width: '50rem', margin: 'auto' }}>
+        <Card.Title>Card Title</Card.Title>
+
+        <div class="latest-photos">
+            <div class="row"><div class="col-md-4"> 
+        <figure> <img class="img-fluid" src="https://i4.hurimg.com/i/hurriyet/75/1110x740/5b8e6d967152d827603dd434.jpg" alt=""/> 
+        </figure></div><div class="col-md-4"> <figure> <img class="img-fluid" src="https://i4.hurimg.com/i/hurriyet/75/1110x740/5b8e6d967152d827603dd434.jpg" alt=""/>
```
Comment (4teko7) [inline Web/src/screens/landingPage.js:26]: With a map function you can create this lines.
let LINKS = [https://i4.hurimg.com/i/hurriyet/75/1110x740/5b8e6d967152d827603dd434.jpg, https://i4.hurimg.com/i/hurriyet/75/1110x740/5b8e6d967152d827603dd434.jpg,]
let FIGURES = []
LINKS.map(res => {
  FIGURES.push(
    <figure>  
  )
})

{FIGURES}

**Review by 4teko7 — CHANGES_REQUESTED** [review]

**Comment by gktpmuhammed** [comment]
I will close this pull request and open a new one. I have corrected some files based on the feedback from Bilal. I have added new functionalities aslo.

**Pull request closed without merging**